### ISSUE TITLE
Adding a new Group fails

### ISSUE THREAD

**Issue by Tobias-Lauerbach** [Bnyro/ConnectYou#72]
### Steps to reproduce

1. Edit Contact
2. Manege group 
3. new
4. selecting new group

same behavior when adding the contact in another App (AOSP or Nextcloud)

### Expected behavior

Group is created an added to the Contact

### Actual behavior

The new Group isn´t saved, and does not show even before Saving the contact..
Group dos not show if created in another Service (only with new Groups, assigning existing ones works fine)

### Connect You version

2.0

### Android version

Android 13 CalyxOS Pixel5

### Other details

thing I tried to solve it:

- close the app
- force stop the app
- reboot
- re-installation 

Herzlichen dank fürs implementieren des features :-)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Groups are only shown if they have any current members. Did you immediately add the contact to the new group by selecting it with the checkbox?

**Comment by Tobias-Lauerbach** [comment]
Yes I have tried that, I also tried also selecting an old group together with the new one, that resulted in only the old one saved.
Both times the groups where added as empty groups to the device (viewable in AOSP)

**Comment by Bnyro** [comment]
What is their account name? (Second option in the filter menu)?

**Comment by Tobias-Lauerbach** [comment]
1. DavX5 sync (nextcloud)

2. same thing if I create a completely new contact and try to add a new label directly (then the contact is created in "Device")

**Comment by HarriBuh** [comment]
I can confirm this bug with contacts stored "device". Adding a new group doesn't work and adding contacts to freshly created groups won't work either.
